e2e_test/test_fee_currency_fails_on_debit.sh

```diff
@@ -3,8 +3,15 @@
 set -eo pipefail
 
 source shared.sh
+source debug-fee-currency/lib.sh
 
 # Expect that the debitGasFees fails during tx submission
-(cd debug-fee-currency && ./deploy_and_send_tx.sh true false false) &>debug-fee-currency/send_tx.log || true
-grep "debitGasFees reverted: This DebugFeeCurrency always fails in debitGasFees!" debug-fee-currency/send_tx.log ||
-  (cat debug-fee-currency/send_tx.log && false)
+(
+	# source ./debug-fee-currency/lib.sh
+
+	fee_currency=$(deploy_fee_currency true false false)
+	# this fails during the RPC call, since the DebitFees() is part of the pre-validation
+	cip_64_tx $fee_currency | assert_cip_64_tx false "fee-currency internal error"
```

Comment:
Yes, it is better to check the response directly instead of grepping through the logs like I did.